lib/ansible/modules/network/nxos/nxos_install_os.py

```diff
@@ -245,6 +245,8 @@ def parse_show_install(data):
             ud['server_error'] = True
         elif data == -32603:
             ud['server_error'] = True
+        elif data == 'No install all data found':
```

Comment:
@mikewiebe there is a check on L241 `if isinstance(data, int)`.
From `data == 'No install all data found'` it looks like data is string. Will the code hit this condition?

Reply:
Good catch.  Your right that it will never hit this condition.  Let me move the check.

Reply:
Turns out this check is not needed.  I removed it.